cmake/os/freertos.cmake

```diff
@@ -28,6 +28,7 @@ set(ETCPAL_OS_ADDITIONAL_SOURCES
   ${ETCPAL_ROOT}/include/os/freertos/etcpal/os_thread.h
 
   ${ETCPAL_ROOT}/src/os/freertos/etcpal/os_lock.c
+  ${ETCPAL_ROOT}/src/os/freertos/etcpal/os_queue.c
```

Comment:
Looks like this got added accidentally from some other work - I think it's failing the build.

Reply:
Yup